### PR TITLE
Add support for iotdin-imx8p device.

### PR DESCRIPTION
**Pull request by vraevsky**

### CODE REVIEW

File: container/flash_iot.sh [hunk 1]
```diff
@@ -41,7 +41,6 @@ cleanup () {
 	exit_code=$?
 	umount $balena_image_boot_mnt > /dev/null 2>&1 || true
 	losetup -d $balena_image_loop_dev > /dev/null 2>&1 || true
-	losetup -D
```
Comment (floion) [inline container/flash_iot.sh:60]: Can you leave a detailed git log message for this commit explaining why this was removed?

File: run_container.sh [hunk 2]
```diff
@@ -51,10 +44,10 @@ while [[ $# -gt 0 ]]; do
 	shift
 done
 
-# if both -d and -i args are passed, run the flasher script in container directly
-if [ ! -z ${dram_conf+x} ] && [ ! -z ${balena_image+x} ]; then
+# if -i arg is passed, run the flasher script in container directly
+if [ [ ! -z ${balena_image+x} ]; then
```
Comment (acostach) [inline run_container.sh:48]: @vraevsky apart from this small typo, this PR looks good to me

Comment (acostach) [reply run_container.sh:48]: and the typo is now fixed in https://github.com/balena-os/iot-gate-imx8plus-flashtools/pull/9

Comment (vraevsky) [reply run_container.sh:48]: @acostach I'd appreciate it if you could point me to the typo.

Comment (floion) [reply run_container.sh:48]: Extra bracket

**Pull request closed without merging**